server.py

```diff
@@ -515,12 +515,40 @@ def post(self):
         backup_helper.recover(location)
         return "Recovery successful."
 
+
+class AssetsControl(Resource):
+    method_decorators = [api_response, auth_basic]
+
+    @swagger.doc({
+        'parameters': [
+            {
+                'name': 'command',
+                'type': 'string',
+                'in': 'path',
+                'description': 'Control command'
+            }
+        ],
+        'responses': {
+            '200': {
+                'description': 'Asset switched'
+            }
+        }
+    })
+    def get(self, command):
+        if command == "next":
+            sh.sudo('systemctl', 'kill', '--signal=SIGUSR1', '--kill-who=main', 'screenly-viewer.service')
```

Comment:
No sudo needed.

Reply:
Also, let's use avoid using the `sh` module for system calls and instead use the [OS library] (https://docs.python.org/3/library/os.html)

Reply:
hmm. it seems that for systemctl calls we still need sudo

Reply:
Why do we need to use `systemctl`?  It's just a regular kill signal. For instance, you could do `pkill -SIGUSR1 -f viewer.py`, but ideally you'd do it in [native python](https://stackoverflow.com/questions/2940858/kill-process-by-name).